mkssins.py

```diff
@@ -0,0 +1,203 @@
+#Imports
+import numpy as np
+import matplotlib.pylab as plt
+import warnings
+import time
+import pickle
+import sys
+import katcali
+import katcali.io as kio
+import katcali.label_dump as kl
+import katcali.diode as kd
+from pathlib import Path
+def good_ant(fname):
+
+    """
+    Fuction to retrieve list of good antennas.
+    """
+
+    data=kio.load_data(fname)
+    bad_ants=kio.check_ants(fname)
+
+    ants_good = []
+    for i in np.array(kio.ant_list(data)):
+        if i not in bad_ants:
+            ants_good.append(i)
+    else:
+        print (str(i) + ' is bad')
+
+    return ants_good
+"""
+Define a function that will access the data block and return visibility data, flags, noise diodes vector. 
+"""
+def visData(fname, ant, pol):
+    """
+    Parameters:
+    ----------
+    Fname: Path to observation block.
+
+    Returns:
+    -------
+    vis, flags (SARAO prior flags), noise_diodes vector (nd)
+
+    Note : Current function looks at one pol and one dish.
+    """
+
+    data = kio.load_data(fname)
+    target, c0, band_ants, flux_model = kio.check_ants(fname)
+    ants_good = good_ant(fname)
+    data.select(ants=ant, pol=pol)
+    recv = ant + pol
+    print(recv)
```

Comment:
You probably won't need this printed out. I suggest adding and option `verbos=False` to the function definition. Then, here you can do
```
if verbos:  # i.e. verbos=True
    print(recv)
```
So you can make it print information for debugging when needed.